VAMobile/src/screens/BenefitsScreen/Letters/LettersOverviewScreen.test.tsx

```diff
@@ -39,11 +36,6 @@ context('LettersOverviewScreen', () => {
     component = render(<LettersOverviewScreen {...props} />, {
       preloadedState: {
         ...InitialState,
-        personalInformation: {
-          ...initialPersonalInformationState,
-          loading: personalInformationLoading,
-          error: personalInformationError ? { networkError: true } : undefined,
-        },
       },
     })
```

Comment:
Another instance of `preloadedState` that can be removed

Reply:
removed